docs/teams/kubernetes.md

```diff
@@ -0,0 +1,50 @@
+# envd on Kubernetes (Experimental)
+
+:::warning
+
+**This feature is in the experimental stage, which means it may be subject to change.**.
+
+:::
+
+`envd` allows you to manage your development environments on Kubernetes. You can use `envd` command to create, delete, and update development environments.
+
+## Run envd-server on Kubernetes
+
+This feature requires a envd-server running on the Kubernetes. Thus you need to deploy it with the helm chart provided in the [envd-server](https://github.com/tensorchord/envd-server) repository:
+
+```bash
+$ git clone https://github.com/tensorchord/envd-server
+$ cd ./envd-server
+$ helm install envd-server ./manifests
```

Comment:
We should publish the charts so users don't have to clone locally. It's pretty easy to set up with Github Actions.

Reply:
Example: https://github.com/akuity/helm-charts

Reply:
SGTM

Reply:
https://github.com/tensorchord/envd-server/pull/73

Reply:
Great!